### DISCUSSION TITLE
Will it be a good idea to use the new DRF Async Views?

### DISCUSSION THREAD

**Discussion by mateokurti** [django/channels#1990] [General]
There has been [some discussion on the DRF repository](https://github.com/encode/django-rest-framework/discussions/7774) about adding support for asynchronous views. The decision (at least as for now) is to support async views through [a third party package, adrf](https://github.com/em1208/adrf). I've tested it out and it works great and I've already made updated my code to be asynchronous for one viewset.

Now my question is, will it be a good idea to use it together with Django Channels? Is there any benefit or drawback on using it?

**Comment by Archmonger** [comment]
There are no drawbacks to using ADRF with Django channels.
